UnitsNet/UnitDefinitions/Flow.json

```diff
@@ -47,6 +59,66 @@
         }
       ]
     },
+    {
+      "SingularName": "CubicFootPerMinute",
+      "PluralName": "CubicFeetPerMinute",
+      "FromUnitToBaseFunc": "x/2118.88000326",
+      "FromBaseToUnitFunc": "x*2118.88000326",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "ft³/min" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "CubicFootPerHour",
+      "PluralName": "CubicFeetPerHour",
+      "FromUnitToBaseFunc": "x*7.8657907199999087346816086183876e-6",
+      "FromBaseToUnitFunc": "x/7.8657907199999087346816086183876e-6",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "cf/hr" ]
```

Comment:
The official abbreviation of cubic foot is ft³ and cf isn't listed on Wikipedia. I would suggest to change this to ft³/h and perhaps add this as an alternative

Reply:
Will do. :)